mt32emu/src/meson.build

```diff
@@ -0,0 +1,63 @@
+mt32emu_config_h_in = files('config.h.in')
+
+common_headers = [
+  'mt32emu.h',
+  'globals.h',
+  'Enumerations.h',
+  'Types.h',
+]
+cpp_headers = [
+  'File.h',
+  'FileStream.h',
+  'MidiStreamParser.h',
+  'ROMInfo.h',
+  'SampleRateConverter.h',
+  'Synth.h',
+]
+c_headers = [
+  'c_interface.h',
+  'c_types.h',
+  'cpp_interface.h',
+]
+
+# These two maps exist only, so we can generate a new directory containing
+# all the headers later on.
+#
+mt32emu_header_map = {}
+foreach header : common_headers + cpp_headers
+  mt32emu_header_map += { header : files(header) }
+endforeach
+mt32emu_c_header_map = {}
+foreach header : c_headers
+  mt32emu_c_header_map += { header : files('c_interface' / header) }
+endforeach
+
+mt32emu_sources = files([
```

Comment:
`files()` does not need to accept one array, it can be `files('one.c', 'two.c')`

Reply:
Thanks! Fixed :)